### PR TITLE
Ps 116 adding .env file

### PR DESCRIPTION
**Pull request by JafariM**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,23 +7,10 @@ import { Navigate, Route, Routes } from 'react-router-dom';
 import Navbar from './components/Navbar';
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
 
-const URL = 'http://localhost:8000/api/v1/';
-
 function App() {
   const [message, setMessage] = useState('');
```
Comment (akosasante) [inline src/App.js:10]: We can remove this as well, it's no longer used here

File: src/App.js [hunk 2]
```diff
@@ -7,23 +7,10 @@ import { Navigate, Route, Routes } from 'react-router-dom';
 import Navbar from './components/Navbar';
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
```
Comment (akosasante) [inline src/App.js:8]: We can also remove `useEffect` and `getAllData` from the imports as they're not used anymore

**Review by akosasante — APPROVED** [review]
LGTM! :+1: 

I changed the base branch to the `connect login` one because this PR seemed to include commits from there. Since we don't want to accidentally merge those until that branch is fixed, I changed the base branch to reflect that for now

File: src/App.js (hunk 2 above)
Comment (JafariM) [reply src/App.js:8]: I removed them

**Comment by JafariM** [comment]
> LGTM! 👍
> 
> I changed the base branch to the `connect login` one because this PR seemed to include commits from there. Since we don't want to accidentally merge those until that branch is fixed, I changed the base branch to reflect that for now

Good, thank you

**Pull request merged**